modules/create_timepoint/test/create_timepointTest.php

```diff
@@ -22,9 +59,40 @@ class createTimepointTestIntegrationTest extends LorisIntegrationTestWithCandida
      */
     function testCreateTimepointDoespageLoad()
     {
-        $this->safeGet($this->url . "/create_timepoint/?candID=000000&identifier=000000");
+        $this->safeGet($this->url . "/create_timepoint/?candID=900000&identifier=900000");
         $bodyText = $this->webDriver->findElement(WebDriverBy::cssSelector("body"))->getText();
         $this->assertContains("Create Time Point", $bodyText);
     }
+
+    /**
+     * Tests that, when loading the create_timepoint module, some
+     * text appears in the body.
+     *
+     * @return void
+
```

Comment:
It would be better to uncomment this test and add a markTestIncomplete() or markTestSkipped()